### PR TITLE
docs: fix site locale

### LINKED ISSUE #37909: table的filter没有重置和确定按钮，官方文档可以直接复现
### Reproduction link

[https://ant.design/components/table-cn/#API 官网直接可以复现](https://ant.design/components/table-cn/#API 官网直接可以复现)

### Steps to reproduce

官方文档table的filter没有重置和确定按钮，应该是跟国际化有关，我之前也碰到过，加上国际化就好了。但是最小复现仓库无法复现，不知道什么原因

这里是antd的代码

image

### What is expected?

根据浏览器语言自动设置一个国际化，然后显示表格filter的重置按钮

### What is actually happening?

表格filter的重置按钮是空白

image

| Environment | Info |
| --- | --- |
| antd | 4.23.4 |
| React | 17 |
| System | macos |
| Browser | chrome 105 |

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/37909

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   -        |
| 🇨🇳 Chinese |   修复文档站 Locale 错误的问题        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by afc163 — APPROVED** [review]

File: site/theme/template/Layout/index.tsx [hunk 1]
```diff
@@ -255,7 +256,7 @@ export default class Layout extends React.Component<LayoutPropsType, LayoutState
             defaultLocale="en-US"
           >
             <ConfigProvider
-              locale={appLocale.locale === 'zh-CN' ? cnLocale : undefined}
+              locale={appLocale.locale === 'zh-CN' ? zhCN : undefined}
```
Comment (zheeeng) [inline site/theme/template/Layout/index.tsx:259]: 这里导致类型不兼容了，antd/lib/locale/zh_CN 下的 Locale 和项目中的 Locale 不兼容

Comment (MadCcc) [reply site/theme/template/Layout/index.tsx:259]: 给个链接？这里是 CP 的 locale 用的应该是 antd 的 locale 而不是 site 的 locale

Comment (zheeeng) [reply site/theme/template/Layout/index.tsx:259]: 本地 lint：

image


VS code 也有这个报错

image


zhCN 是 'antd/lib/locale/zh_CN' 引入的，它是 4.21.7 的 Locale

Comment (MadCcc) [reply site/theme/template/Layout/index.tsx:259]: bisheng 有 alias，'antd/lib/*' 实际上指向的是 'components/*'。
CI 都是通过的，可以看下你本地有什么配置不一样。

Comment (zheeeng) [reply site/theme/template/Layout/index.tsx:259]: 类型的问题那应该是 tsconfig 要配一下 alias 了

**Pull request merged**